src/concrete/ml/pytest/torch_models.py

```diff
@@ -1379,9 +1381,9 @@ def forward(self, x):
         x_pre = []
 
         for i in range(self.n_blocks):
-            x_block = x[:, i, :]
-            q1_out = self.quant_1(x_block)
-            fc1_out = self.fc1(q1_out)
+            q_x = self.quant_1(x)
```

Comment:
this was wrong.. the input to the slicing should be quantized with QuantIdentity